refocus-bdk-client.js

```diff
@@ -677,6 +678,17 @@ module.exports = (config) => {
       return genericPost(`${SERVER}${API}${EVENTS_ROUTE}`, events, TOKEN);
     }, // createEvents
 
+    /**
+     * Create multiple events at once
+     *
+     * @param {Array} events - Array of events to be created
```

Comment:
@ConalCosgrove Maybe it's worth putting a line under this showing what an event object should look like. I think there is just one required field with a key 'log'